File: C++/archivos cpp/make_box.cpp
#include<iostream>
#include<vector>
using namespace std;

void make_box(int num){ //num = 3
	vector<string> box;
	string v, temp, spaces;
	// First Row
	for(int i=0; i<num; i++){
		v+='#';
	}
	box.push_back(v);
	temp = v;
	// Spaces between columns
	for(int i=0; i<num-2; i++){
		spaces+=" ";
	}
	// Two Columns
	for(int i=0; i<num-2; i++){	
		v = '#' + spaces + '#';
		box.push_back(v);
	}
	// Adding end row
	box.push_back(temp);

	// Showing the box
	for(auto& i: box){
		cout<<i<<endl;
	}
}


int main(int argc, char const *argv[]){
	make_box(30);
	return 0;
}

File: Python/Práctica/make_box.py

def make_box(num):
	box = []
	v = ""
	spaces = ""
	for i in range(num):
		v+="#"
	box.append(v)
	temp = v
	v = ""
	for i in range(num-2):
		spaces+=" "

	for i in range(num-2):
		v = "#" + spaces + "#"
		box.append(v)
	box.append(temp)

	for c in box:
		print(c)

make_box(4)